api/controllers/Cid_EntryController.js

```diff
@@ -71,6 +71,12 @@ module.exports = {
       .populate('female_active_movements_in')
       .populate('female_active_movements_out')
       .then(centres => _.map(centres, centre => Centres.publishUpdate(centre.id, centre)))
+      .then(() => {
```

Comment:
err?
https://github.com/UKHomeOffice/removals_integration/pull/127/files#diff-d745622fd2ed9aaf163c461cc3e081baR84
you're returning a promise there.
and also the real one in waterline returns a promise